.github/workflows/merge-master.yml

```diff
@@ -41,32 +40,58 @@ jobs:
       id: version
       run: echo "::set-output name=value::${{ steps.semantic.outputs.new_release_version }}.rc${{ steps.build_number.outputs.value }}"
 
-  prerelease:
-    if: needs.prepare.outputs.new_release == 'true'
-
-    name: Generates Prerelease
+  docs-changelog:
+    name: Update changelog on docs 
     runs-on: ubuntu-20.04
 
     needs: prepare
 
     steps:
-    - name: Create Pre Release
-      uses: actions/create-release@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Calculate changelog path
+      id: path
+      run: echo "$(echo "::set-output name=value::docsrc/source/pages/reference/changelog/${{ needs.prepare.outputs.release }}" | sed "s/\./_/g").rst"
+
+    - name: Configure Git
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git config --local core.autocrlf false
+    
+    - name: Convert changelog from md to rst
+      run: |
+        sudo apt-get install -y pandoc
+        echo "${{ needs.prepare.outputs.release_notes }}" \
+        | sed 's/# \[.*/# Changelog v3.4.0/g' \
```

Comment:
we should remove this hardcoded 3.4.0, right? 🤔

Reply:
yes, right, my mistake, thank you!